compiler/optimizer/LocalOpts.cpp

```diff
@@ -6932,182 +6939,136 @@ int32_t TR_InvariantArgumentPreexistence::perform()
 
       if (argInfo)
          {
-         if (trace())
+         if (enableTrace)
             traceMsg(comp(), "PREX:    Populating parmInfo of inlined method %s from argInfo %p\n", feMethod->signature(trMemory()), argInfo);
 
          TR_PrexArgInfo *argInfo = comp()->getCurrentInlinedCallArgInfo();
```

Comment:
can this line be deleted?

Reply:
@liqunl can you respond to this comment? then I think we can get this merged after testing.

Reply:
Done